app/views/manage_submissions/_submissions_table.html.erb

```diff
@@ -0,0 +1,71 @@
+<table class="usa-table usa-table--stacked-header usa-table--borderless width-full">
+    <thead>
+      <tr>
+        <th scope="col">Submission ID</th>
+        <th scope="col">Eligible for Evaluation</th>
+        <th scope="col">Selected to Advance</th>
+        <th scope="col">Assigned Evaluators</th>
+        <th scope="col">Average Score</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @submissions.each do |submission| %>
+          <tr>
+            <th data-label="Submission ID" scope="row">
+              <span class="mobile-lg:display-none">Submission ID </span><%= submission.id %>
+            </th>
+            <td data-label="Eligible for Evaluation">
+              <div class="display-flex flex-align-center">  
+                <% if eligible_for_evaluation?(submission) %>
+                    <%= image_tag(
+                        "images/usa-icons/verified.svg",
+                        class: "usa-icon--size-3",
+                        alt: ""
+                    )%>
+                    Eligible for Evaluation
+                <% else  %>
+                    <%= image_tag(
+                        "images/usa-icons/highlight_off.svg",
+                        class: "usa-icon--size-3",
+                        alt: ""
+                    )%>
+                    Not Eligible for Evaluation
+                <% end %>
+              </div>  
+            </td>
+            <td data-label="Selected to Advance">
+              <div class="display-flex flex-align-center">  
+                <% if selected_to_advance?(submission) %>
+                    <%= image_tag(
+                        "images/usa-icons/star.svg",
+                        class: "usa-icon--size-7 icon-white margin-x-2",
+                        alt: ""
+                    )%>
```

Comment:
the size is off and it seems like it should be a filled in star instead of light grey
![Screenshot 2024-11-04 at 12 52 26 PM](https://github.com/user-attachments/assets/2a840073-fceb-4aac-b4f3-5424e0abf48a)

```suggestion
                    <%= image_tag(
                        "images/usa-icons/star.svg",
                        class: "usa-icon--size-3 margin-x-2",
                        alt: ""
                    )%>
```